### ISSUE TITLE
Glamor benchmarks are broken (missing CSS)

### ISSUE THREAD

**Issue by rtsao** [hellofresh/css-in-js-perf-tests#34]
Running the benchmarks (with fresh node_modules), for the simple case the resulting size is `244 B  (155 B gzipped)` which corresponds to:

```
<html>
    <head>
        <style type="text/css">

        </style>
    </head>
    <body>
<section class="css-h433f4">
    <h1>=========== glamor ===========</h1>
    <button class="css-wboq2z">Click me</button>
</section>
    </body>
</html>
```

@threepointone I'm guessing this has something to with your WeakMap cache implementation?

**Comment by threepointone** [comment]
good catch, looks like my weakmaps don't defeat these tests after all, `flush()` still clears css so the rule isn't that useful (yay for mutable state). 

published 2.20.20 with the fix.

**Comment by rtsao** [comment]
Yep, 2.20.20 seems to be working!
